pkg/addons/default/generate.go

```diff
@@ -1,3 +1,5 @@
 package defaultaddons
 
+//go:generate curl --silent --location https://github.com/aws/amazon-vpc-cni-k8s/blob/dd631108e61a977809f9b1a1c40232637e734184/config/v1.5/aws-k8s-cni.yaml?raw=1 --output assets/aws-node.yaml
```

Comment:
I can see this was remove in 947cff2907a54ee7e33d15f7cf6432bd76f8a4f2 and it shouldn't had been. Thanks for fixing!